harper-core/dictionary.dict

```diff
@@ -8483,7 +8482,7 @@ Ritz/2M
 Rivas/2M
 Rivera/2M
 Rivers/2M
-Riverside/2M
+Riverside/2M@
```

Comment:
A bunch of what look like generic place names and random words are getting marked as Canadian `@` - is this right? (See also `Vallejo`, `Villa`, `abridge`, etc) - maybe I just can't see the pattern yet?